frontend/src/components/slc/components/functions-table/components/action-cell/styles.module.scss

```
@@ -1,4 +1,5 @@
 .actionsWrapper {
```

Maybe change to wrapper like other files?
```suggestion
.wrapper {
```